app/views/about/index.html.erb

```diff
@@ -16,105 +15,28 @@
-              </tbody>
-            </table>
-          <% end %>
+          <%= data = @smart_proxies.map {|proxy| {:id => {:name => proxy.name, :id => proxy.id},
+                                                  :features => h(proxy.features.map(&:name).to_sentence)}} %>
+          <%= mount_react_component('AboutProxyTable', '#smart_proxies', data.to_json) %>
         </div>
         <% if SETTINGS[:unattended] %>
           <div class="tab-pane" id="available_providers">
-            <table class="table table-striped">
-              <thead>
-                <tr>
-                  <th><%= _("Provider") %></th>
-                  <th><%= _("Status") %></th>
-                </tr>
-              </thead>
-              <tbody>
-              <% @providers.sort_by { |prov| prov[:friendly_name].downcase }.each do |provider| %>
-                <tr>
-                  <td><%= provider[:friendly_name] %></td>
-                  <% if provider[:status] == :installed %>
-                    <td><div class="label label-success"><%= _('Installed') %></div></td>
-                  <% else %>
-                    <td><div class="label label-default"><%= _('Not Installed') %></div></td>
-                  <% end %>
-                </tr>
-              <% end %>
-              </tbody>
-            </table>
+            <%= data = @providers.map {|provider| {:provider => provider[:friendly_name],
+                                                   :status => provider[:status] == :installed}}%>
+            <%= mount_react_component('AboutProviderTable', '#available_providers', data.to_json) %>
+          
             <%= _('To enable a provider, either install the OS package (e.g. foreman-libvirt) or enable the bundler group for development setup (e.g. ovirt).') %>
           </div>
           <div class="tab-pane" id="compute_resources">
-            <% if @compute_resources.empty? %>
-              <p class="ca"><%= _("No compute resource to show") %></p>
-            <% else %>
-              <table class="table table-striped table-fixed">
-                <thead>
-                  <tr>
-                    <th class="col-md-4"><%= _("Name") %></th>
-                    <th class="col-md-4"><%= _("Type") %></th>
-                    <th class="col-md-4"><%= _("Status") %></th>
-                  </tr>
-                </thead>
-                <tbody>
-                <% @compute_resources.each do |compute| %>
-                  <tr>
-                    <td class="ellipsis"><%= link_to(compute.name, compute) %></td>
-                    <td><%= compute.provider_friendly_name %></td>
-                    <td><div class="compute-status" data-url=<%= ping_compute_resource_path(compute) %>><%= spinner %></div></td>
-                  </tr>
-                <% end %>
-                </tbody>
-              </table>
-            <% end %>
+            <%= data = @compute_resources.map {|compute| {:id => {:name => compute.name, :id => compute.id},
```

Comment:
when there are none, you get a generic message ````No data found``` - we should use the PF empty state instead.